### PR TITLE
Add option to stay in current view after render

### PR DESCRIPTION
**Pull request by LeonardMeagher2**
This PR just adds an option to have the AI rendered result go to the Image editor that your blender render result went to instead of opening up the AIR workspace.

The main reason I'm hoping to get this feature is I have a workflow for using the AI rendered images to texture object, with my previous PR #133 I can keep the image name consistent, and with this I can stay in my view, speeding up my process a lot. Making it all optional is to make sure the vision you have for your workflow can stay the same.

### CODE REVIEW

**Comment by benrugg** [comment]
Yes! This is honestly much better than switching to a new workspace. In the earlier versions of AI Render, I created the workspace and also had a node setup, because I didn't know how to load the image back in. When I changed it, I would have changed it to do exactly this if I had thought through it better.

I think this new version should actually replace the way I am currently doing it, and it shouldn't even be an option. We can even get rid of the AI Render workspace altogether.

Would you like to take a pass at doing that??

**Comment by LeonardMeagher2** [comment]
Absolutely!

**Comment by LeonardMeagher2** [comment]
@benrugg I've removed the air workspace

File: operators.py [hunk 1]
```diff
@@ -123,20 +94,15 @@ def ensure_animated_prompts_text():
         text.select_set(0, 3, 0, -1)
 
 
-def ensure_animated_prompts_text_editor_in_workspace(context):
-    script_area = utils.get_area_by_type('TEXT_EDITOR', workspace_id=config.workspace_id)
+def ensure_animated_prompts_text_editor(context):
```
Comment (LeonardMeagher2) [inline operators.py:97]: I haven't actually used animated prompts, so either you can test that, or I can figure it out haha

Comment (LeonardMeagher2) [reply operators.py:97]: Okay found it quickly and this breaks, I'll fix it

Comment (LeonardMeagher2) [reply operators.py:97]: this is fixed now

**Comment by benrugg** [comment]
This is great work. When I asked if you wanted to do it, I honestly didn't think through how much work it would be. You had to touch a lot, and you did it expertly - thank you!

**Comment by benrugg** [comment]
I'll probably wait to release this until we do both PR's at once.

**Pull request merged**